pkg/operator/ceph/object/rgw.go

```diff
@@ -69,7 +69,7 @@ func (c *config) createOrUpdate(update bool) error {
 	if err == nil && exists {
 		if !update {
 			logger.Infof("object store %s exists in namespace %s", c.store.Name, c.store.Namespace)
-			return nil
+			return c.startRGWPods(false)
```

Comment:
Shouldn't it be `true` then that an update will be made if it exists?

Reply:
The `update` flag is intended to indicate if the crd was just created or if the crd was just updated. If the crd was updated, it will already fall through to the rest of this method and perform the update. So this change will ensure that the update is performed during the "create" crd action, which is triggered every time the operator starts even though it previously existed.

Reply:
I think `false` is the right thing here. It may actually be best to clean up this function so it doesn't take an `update` param since I believe that logic can be assumed by `UpdateDeploymentAndWait`